core/storage/feedback/gae_models.py

```diff
@@ -271,8 +377,19 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
                state_name, description, state_content):
         """Creates a new SuggestionModel entry.
 
-        Throws an exception if a suggestion with the given thread id already
-        exists.
+        Args:
+            exploration_id: str. ID of the corresponding exploration.
+            thread_id: str. ID of corresponding thread.
+            author_id: str. ID of user who submitted the suggestion.
+            state_name: str. ID of the state the thread is for.
+            description: str. Learner provided description of suggestions.
+            state_content: dict. Contains two keys, "type" and "value".
```

Comment:
Can you please explain which action involves this method ? Because whenever I am sending feedback in any exploration, only log messages (that I have written) of generate_new_thread_id methods are displayed.